cves/CVE-2013-0848.yml

```
@@ -89,21 +91,23 @@ fixes_vcc_instructions: |
   The notes field is optional - place anything that clarifies things or you
   find interesting in there.
 fixes:
-- commit: 6abb9a901fca27da14d4fffbb01948288b5da3ba
-  note: ''
-- commit: a7153444df9040bf6ae103e0bbf6104b66f974cb
-  note: ''
-- commit: ''
-  note: ''
+  commit: 6abb9a901fca27da14d4fffbb01948288b5da3ba
+  note: 'check width more completely, avoid out of array accesses'
```

```suggestion
  note: check width more completely, avoid out of array accesses
```
I don't think quotes are necessary here.